spec/spec_helper.rb

```diff
@@ -124,6 +124,7 @@ def mock_user(attributes = {})
     groups: [],
     can_view_something?: false,
     is_admin?: false,
+    is_webauth_admin?: attributes[:is_admin?],
```

Comment:
Yes, it was added in https://github.com/sul-dlss/argo/commit/0c2196439fbae8a942c3289dc039fda1fe8816f8